app/views/includes/phase_banner.html

```diff
@@ -1,6 +1,6 @@
 <div class="phase-banner">
   <p>
     <strong class="phase-tag">ALPHA</strong>
-    <span>This is a new service – your <a href="https://designpatterns.hackpad.com/GOV.UK-elements-feedback-sKrDyQxcfA2">feedback</a> will help us to improve it.</span>
```

Comment:
I don't think this is correct. The original link went to something about feeding back in general, not specifically for the phase banners. As this include isn't used anywhere right now, I would probably just exchange the link with a `#` so that a new link can be added whenever more feedback is required.

Reply:
replaced with #